### DISCUSSION TITLE
distribute oh-my-posh as a fisher module on fish shell

### DISCUSSION THREAD

**Discussion by mess-maker** [JanDeDobbeleer/oh-my-posh#2134] [Troubleshoot]
hi,

don't know if that's possible.
have you considered delivering oh-my-posh as a fish shell plugin via fisher (https://github.com/jorgebucaran/fisher)
this could bring the same seamless installation as modules on windows terminal

cheers

**Comment by JanDeDobbeleer** [comment]
well, I'm going to deprecate the module for a ton of reasons, this will eventually be the same struggle. Even if we can, the answer is no as I can't afford all the extra effort required to maintain this.

**Comment by mess-maker** [comment]
ok I understand
So is there a simple way to update oh-my-posh after install on linux for brew-less distro ?
or i'm missing something ?

**Reply by JanDeDobbeleer** [reply]
That's a good question. Maybe a simple `install.sh` script to execute could do the trick.

**Reply by mess-maker** [reply]
sure
but after a manual primary install
a `oh-my-posh self-update` like command would be ace
